src/Core/Language/LanguageRepositoryInterface.php

```diff
@@ -33,16 +33,31 @@
 interface LanguageRepositoryInterface
 {
     /**
+     * Returns a LanguageInterface which locale matches the provided one.
```

Comment:
```suggestion
     * Returns a LanguageInterface which locale match the provided one.
```
?

Reply:
Mmm, there's only one locale matching so it's singular no?

Reply:
Or do you think it should be called `getOneByLocale` then? (same for other methods of course)

Reply:
we're returning only one element, and not and array, this is why I was thinking about singular :/

Reply:
Yes it's singular hence the `matches`, plural would have been `which locales match`
That said this interface is new for 176 so we can still rename it, I you think `getOneByLocale` is more appropriate?

Reply:
why not :)

Reply:
Done